### ISSUE TITLE
ToDo app as Prework for codepath android bootcamp

### ISSUE THREAD

**Issue by ipande** [ipande/SimpleToDoList#3]
Please take a look at my submission, @codepathreview @codepath.
As i mentioned in the notes, i felt like adding a SQLite db for persistence was an overkill. 
(Basically we're not using any powerful CRUD features of SQL). I had initially implemented 
persistence using sharedPrefs. Your thoughts??

**Comment by nesquena** [comment]
Ishan,

Looks good, this exercise is intended in part to give you an introduction to the general rhythm of this course. The course is entirely project based with an app being assigned each week and then due the following week. Each project builds on the last to help each engineer to learn all practical Android development and best practices as quickly as possible. We also do a code review for each submitted project.

The next step is to continue working on [extensions to your todo app](http://courses.codepath.com/snippets/intro_to_android/prework#heading-5-extending-your-todo) and to **schedule a short 10 minute phone conversation** with us. Navigate to [today's date and onward from there](https://calendar.google.com/calendar/selfsched?sstoken=UURvT01vbzBza1pnfGRlZmF1bHR8NzVjYzBkN2EyZTRiOGRmYjZlZjhmMjA2N2IyMGMwYWU) and choose a 15-minute slot. Let us know if none of those times work. 

With regards to extensions to add, we are looking for developers with an interest in user experience and product development so be sure to focus on additional functional enhancements and/or improving the user interface of the app.

Once you select a slot, can you make sure to **include best number** to reach you at to the invite? Look forward to chatting soon!

**Comment by nesquena** [comment]
> As i mentioned in the notes, i felt like adding a SQLite db for persistence was an overkill. 
> (Basically we're not using any powerful CRUD features of SQL). I had initially implemented 
> persistence using sharedPrefs. Your thoughts??

I would agree SQL would be overkill for the basic app as it is now although we'd like to see you add additional attributes to the todo item such as due date, priority and color creating a fuller model object. You could also add additional features (sorting, filtering) in which SQL would become more appropriate.
